examples/Makefile-nrf52840

```diff
@@ -79,6 +79,10 @@ COMMONCFLAGS                   := \
     -I$(CONFIG_FILE_PATH)         \
     $(NULL)
 
+ifeq ($(CERT_LOG),1)
+COMMONCFLAGS += -DOPENTHREAD_CONFIG_ENABLE_DEFAULT_LOG_OUTPUT=1
```

Comment:
I tried to use common-switches.mk but there is no --enable-default-logging configure option anymore. Because of that common-switches.mk does not work with OPENTHREAD_CONFIG_ENABLE_DEFAULT_LOG_OUTPUT option.